providers/gce/support.go

```diff
@@ -34,7 +34,7 @@ type gceProjectRouter struct {
 // ProjectID returns the project ID to be used for the given operation.
 func (r *gceProjectRouter) ProjectID(ctx context.Context, version meta.Version, service string) string {
 	switch service {
-	case "Firewalls", "Routes", "Subnetworks":
+	case "Firewalls", "Routes", "Subnetworks", "Networks":
```

Comment:
is this ok, it seems a change in behavior

Reply:
It is a change in behaviour. Basically what we are saying is that for calls that are from the 'Networks' service, we need to use the network project. This was a bug earlier.

Reply:
got it, thanks